lib/gui/app.js

```diff
@@ -100,9 +101,12 @@ app.run((AnalyticsService, ErrorService, UpdateNotifierService, SelectionStateMo
     currentVersion: packageJSON.version
   });
 
-  UpdateNotifierService.isLatestVersion().then((isLatestVersion) => {
+  const CURRENT_VERSION = ManifestBindService.get('version');
 
-    if (isLatestVersion) {
+  s3Packages.getLatestVersion(s3Packages.BUCKET_URL_PRODUCTION, {
+    currentVersion: CURRENT_VERSION
+  }).then((latestVersion) => {
+    if (latestVersion === CURRENT_VERSION) {
```

Comment:
Sounds reasonable. What do we return from `getLatestVersion` if we hit a network issue, for example? I guess we can return `null` and let the user check if that's the case when doing to semver comparison.